applications/connectivity_bridge/boards/thingy91_nrf52840.conf

```diff
@@ -28,3 +28,14 @@ CONFIG_USB_DEVICE_PRODUCT="Thingy:91 UART"
 
 CONFIG_USB_DEVICE_VID=0x1915
 CONFIG_USB_DEVICE_PID=0x9100
+
+CONFIG_USB_MAX_NUM_TRANSFERS=8
+CONFIG_BRIDGE_CMSIS_DAP_BULK_ENABLE=y
+
+CONFIG_DAP=y
+CONFIG_DP_DRIVER=y
+
+CONFIG_CMSIS_DAP_DEVICE_VENDOR="Nordic Semiconductor ASA"
+CONFIG_CMSIS_DAP_DEVICE_NAME="nrf91"
+
+CONFIG_HEAP_MEM_POOL_SIZE=6000
```

Comment:
RAM is tight - had to decrease usage.

Reply:
FYI: A lot of RAM is used for the USB CDC FIFOs. In prj.conf, `CONFIG_USB_CDC_ACM_RINGBUF_SIZE=16384`. As there are 2 x CDC instances, there are 4 of these buffers in total (=64 KiB RAM). So only a slight reduction of this parameter could give you the RAM you need,
Sadly the "old" USB CDC class doesn't let you set the buffer size on a per instance basis. The USB-next version will. I suspect only the CDC instance used for trace would need this kind of buffering.

Reply:
alright, I slightly reduced the cdc buffer instead

Reply:
I hope the missing 1K isn't going to cause any trouble with certificate provisioning. There still is a lot of buffer.